plugins/@grouparoo/airtable/src/lib/appTest.ts

```diff
@@ -0,0 +1,11 @@
+import { TestPluginMethod } from "@grouparoo/core";
```

Comment:
convention is to name this file `test` (the method name should be the same, as that's the part of the plugin interface that it is implementing)